lib/open_project/translations/models/locales_updater.rb

```diff
@@ -107,13 +106,18 @@ def download_and_replace_locales
       language_name = entry.name.split('/').first # the file is put in a directory containing the language name
 
       # only take translations with enough percent translated
-      next unless @i18n_provider.translation_status_high_enough?(language_name, ACCEPTANCE_LEVEL)
+      next unless @i18n_provider.translation_status_high_enough?(language_name, acceptance_level)
 
       filepath = target_directory.join "#{js_translation?(Pathname.new(entry.name)) ? 'js-' : ''}#{language_name}.yml"
       replace_file(filepath, entry)
     end
   end
 
+  def acceptance_level
+    return 100 if ENV['ACCEPTANCE_LEVEL'].nil?
+    ENV['ACCEPTANCE_LEVEL'].to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.